### ISSUE TITLE
Review about the Test

### ISSUE THREAD

**Issue by sinansevgi** [Bishoy-Samwel/js-capstone#32]
Hello @Bishoy-Samwel, @sheylaPozo, and @2bleO 

This weekend, I approved your project because it was about the deadline for submission. But I would like to raise an issue about your test file to just clear things. 

We usually use the mocks to mock API responses (or other external dependencies like databases), not for the parts of our application. 
You have created a whole new class in your [comments.js](https://github.com/Bishoy-Samwel/js-capstone/blob/develop/src/__mocks__/comment.js) mock file and this class has nothing to do with your original class. 
Basically in your test file, you are testing the class that you created in your mock file, not the original class. 

You can check [this](https://www.loupetestware.com/post/mocking-api-calls-with-jest) article to learn about mocking the API calls. 

You don't have to do any changes to your project, I just wanted to create this issue to inform you. :raised_hands:

**Comment by Bishoy-Samwel** [comment]
Thank you @sinansevgi, I'll look into the article, and I'll implement the best practices.
